pkg/components/pluggable/errors.go

```diff
@@ -113,7 +113,7 @@ func KitErrorFromStatus(s *status.Status) (error, bool) {
 		return s.Err(), false
 	}
 
-	return kiterrors.NewBuilder(s.Code(), httpCode, s.Message(), info.GetMetadata()[tagField]).
+	return kiterrors.NewBuilder(s.Code(), httpCode, s.Message(), info.GetMetadata()[tagField], "unknown").
```

Comment:
We need to look a bit deeper into this, if I remember correctly how pluggable components work, we won't be able to extract the original category, but we could maybe create a new generic one, for example, `pluggable-state`.  @JoshVanL what do you think?
For context (so you don't have to look through the whole PR), we added a `category` field needed for recording metrics to kit and APIErrors, but we never return it as part of the final error the client sees. Does that mean a pluggable component won't have access to it?

Reply:
I'm also ok marking this as category `pluggable-component` for now, and split it in the future if necessary.